### PR TITLE
Fixes #59 - missing error for hammer-config file

### LINKED ISSUE #59: Cannot setup foreman_maintain, complains about foreman-maintain-hammer.yml not found
Looking at the https://github.com/iNecas/foreman_maintain/tree/master/config , I can see just two config files  foreman_maintain.yml.example,hammer.yml.example.
When i run the foreman-maintain have the setting configured
```

./foreman-maintain health check
 
Running checks with tags [default]
================================================================================
check for paused tasks:                                               [FAIL]
There are currently 2 paused tasks in the system
--------------------------------------------------------------------------------
There are multiple steps to proceed:
1) resume paused tasks
2) investigate the tasks via UI
Select step to continue, [n(next), q(quit)] 1
resume paused tasks:                                                  [OK]
Error: Custom configuration file /root/foreman_maintain/config/foreman-maintain-hammer.yml does not exist.
--------------------------------------------------------------------------------
Rerunning the check after fix procedure
check for paused tasks:                                               [FAIL]
There are currently 2 paused tasks in the system
```

Am I missing something?

### PR DESCRIPTION
**Pull request by kgaikwad**
As we already have foreman-tasks feature, moved hammer code from resume procedure to feature file.
Which will ask hammer credentials instead of showing error message for missing config file.

### CODE REVIEW

File: definitions/features/foreman_tasks.rb [hunk 1]
```diff
@@ -82,6 +82,10 @@ def condition(state)
     end
   end
 
+  def resume_task_using_hammer
+    hammer('task resume')
```
Comment (iNecas) [inline definitions/features/foreman_tasks.rb]: Don't we need to include hammer module here as well?

Comment (kgaikwad) [inline definitions/features/foreman_tasks.rb]: @iNecas
Hammer module is already included in `lib/foreman_maintain/feature.rb`. That's why we don't need to include hammer module here.

Comment (iNecas) [inline definitions/features/foreman_tasks.rb]: Oh, you're right. Thanks for explanation.

**Comment by iNecas** [comment]
Thanks @kgaikwad

**Pull request merged**